src/main/java/backend/IssueMetadata.java

```diff
@@ -12,18 +12,26 @@
     private final List<TurboIssueEvent> events;
     private final List<Comment> comments;
     private final LocalDateTime nonSelfUpdatedAt;
+    private final LocalDateTime selfUpdatedAt;
     private final int nonSelfCommentCount;
+    private final int selfCommentCount;
 
     // If isUpdated is true, nonSelfUpdatedAt will be used to sort/filter instead of updatedAt in TurboIssue
     private final boolean isUpdated;
+    private final boolean isUpdatedBySelf;
+    private final boolean isUpdatedByOthers;
```

Comment:
So `isUpdated` tells us whether or not metadata is up to date in general, and the other two tell us whom the last update was by?

Maybe you could add a brief comment explaining the purpose of each field. Without context it can be difficult to figure out what they are for.